packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -145,6 +145,7 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
       showLabel={!compactMode}
       styles={controlButtonBaseStyle}
       {...microphoneButtonStrings}
+      allowDisabledFocus={true}
```

Comment:
nit: thoughts on having this property as default on the ControlBarButton.tsx:
```
return (
    <ControlButtonTooltip content={tooltipContent} id={tooltipId}>
      <DefaultButton
          allowDisabledFocus={props.allowDisabledFocus ?? true}
```
?
Pros: accessibility by default for customers
Cons: slight divergence from Fluent's default button

Reply:
I was wondering about it too but decided to not go there as it should be a decision on Contoso when using components.
I agree that having it as true by default is a better behaviour, but that's something I can discuss with Alex P to have a better insights

Reply:
Contoso can still decide, we'd only be setting allowDisabledFocus to be true _by default_ (they can set it to false if they don't want it). I'd still advocate for having it in the control bar button as we claim to be an a11y-centric library in our storybook but then wouldn't be applying a11y specific properties up front (Contoso has to set a quite hidden property to get the a11y feature)

Reply:
true :)

Reply:
I will update the storybook doc to reflect this though